src/researchhub_document/filters.py

```diff
@@ -20,24 +29,37 @@
     ("posts", "Posts"),
     ("hypothesis", "Hypothesis"),
     ("question", "Questions"),
+    ("bounty", "Bounty"),
 )
-BOUNTY_CHOICES = (
-    ("all", "All"),
-    ("none", "None")
-    # ("open", "Open"),
-    # ("cancelled", "Cancelled"),
-    # ("expired", "Expired"),
-    # ("closed", "Closed"),
-)
-UNIFIED_DOCUMENT_FILTER_CHOICES = (
-    ("removed", "Removed"),
-    ("top_rated", "Top Rated"),
-    ("most_discussed", "Most Discussed"),
-    ("newest", "Newest"),
-    ("hot", "Hot"),
+TAG_CHOICES = (
```

Comment:
Wonder if there's a world where these tags & filter options are auto-synced with FE. 
Just to avoid the hassle of going back&forth to bugfix make updates to these in the future.

Reply:
That would be something nice to have. I'm not sure how we would implement something like that - maybe some sort of file hosted on S3 that we can grab the variables from both the BE and FE?